mimic/model/queue_objects.py

```diff
@@ -0,0 +1,61 @@
+"""
+Models for the Queue plugin
+"""
+
+from __future__ import absolute_import, division, unicode_literals
+
+import attr
+from six import text_type
+
+from mimic.util.helper import random_hex_generator
+
+
+@attr.s
+class Queue(object):
+    """
+    A Queue object in Cloud Queues.
+    """
+    name = attr.ib(validator=attr.validators.instance_of(text_type))
+    id = attr.ib(validator=attr.validators.instance_of(text_type),
+                 default=attr.Factory(lambda: random_hex_generator(4)))
+    _messages = attr.ib(default=attr.Factory(list))
+
+    def brief_json(self):
+        """
+        A brief representation of this queue that can be serialized
+        via json.dumps.
```

Comment:
This generates "brief" JSON that shows up in a specific place in the API; it might be nice to document that.